packages/block-editor/src/components/block-draggable/style.scss

```
@@ -1,6 +1,9 @@
 .block-editor-block-draggable-chip-wrapper {
 	position: absolute;
-	top: -$block-toolbar-height - $grid-unit-15;
+	// Offset position by half the height of the draggable chip
+	// so that the cursor is centered over the left part of the chip.
+	top: -$grid-unit-30;
+	left: -$grid-unit-30;
```

Wouldn't it be better if we calculate it based on `$block-toolbar-height`? So that when we change the value of it, this can also be changed accordingly.

```suggestion
	top: -#{$block-toolbar-height / 2};
	left: -#{$block-toolbar-height / 2};
```

---

Ah, yeah, good idea, I missed that the chip itself has `height: $block-toolbar-height;`. 😄